FluentUI.Demo/src/androidTest/java/com/microsoft/fluentuidemo/demos/V2PeoplePickerUITest.kt

```diff
@@ -53,7 +54,7 @@ class V2PeoplePickerUITest {
             })
         }
         composeTestRule.onRoot().printToLog("testPeoplePickerChip")
-        val textField = composeTestRule.onNodeWithTag("Text field")
```

Comment:
localize the Tag

Reply:
These strings are local to the test class and not used outside, should we localize these strings? I think these tests wont fail even of the language is set to non english